src/showcase.tsx

```diff
@@ -322,6 +323,7 @@ export class Index extends React.Component<any, any> {
                     width={330} 
                     height={200} 
                 ></LineChart>
+                <ProgressBar title={'RAM'} width={400} height={20} data={{total: 15999, current: 12560}} percentage={'79 %'}></ProgressBar>
```

Comment:
IUsage is used so that internally component can calculate the size of progress and percentage is used for displaying it alongside progress. You're right both values seem a bit redundant, so I could either pass a total and current values and calculate percentage in class or just pass percentage.